src/lib/index.ts

```diff
@@ -26,23 +26,23 @@ const main = async () => {
     let keepHistory = false;
     let setPassIfNoBaselineFlag = false;
     let detailsLink = '';
+    let failOn = undefined
 
     const options = process.argv.slice(8)
 
     options.forEach(option => {
-      if (option === 'keepHistory')
-      {
+      if (option === 'keepHistory') {
         keepHistory = true
-      } else if (option === 'setPassIfNoBaselineFlag')
-      {
+      } else if (option === 'setPassIfNoBaselineFlag') {
         setPassIfNoBaselineFlag = true
-      } else
-      {
+      } else if(option === 'upgradable' || option === 'patchable' || option === 'all') {
```

Comment:
Why are we not looking for failOn rather than the options' values?
For example: in keepHistory we look for the option name and not value.

Reply:
because keepHistory or the other one are flags. We don't have a clean way to get something failOn=all without doing parsing and so on. If we want that, I'd rather refactor the whole thing, but the breaking change it would introduce makes me sad and I don't have the bandwidth to deal with that at the moment, so I'm putting it off.

Reply:
Could you share an example of passing this new parameter? I can't see it inthe docs and a little unclear from the test